templates/partials/candidates-office-table.html

```diff
@@ -0,0 +1,17 @@
+<table id="results" class="data-table" data-type="candidate">
```

Comment:
What do you think about moving the `*-table.html` partials to just include them in the main page template for these pages? Both don't have that much code, and we never reference them anywhere else, so it could cut down on the number of files you need to create for each page.